jerry-core/parser/js/js-parser-module.c

```diff
@@ -251,12 +249,12 @@ parser_module_parse_export_clause (parser_context_t *context_p) /**< parser cont
     }
 
     lexer_literal_t *literal_p = PARSER_GET_LITERAL (local_name_index);
-    local_name_p = ecma_new_ecma_string_from_utf8 (literal_p->u.char_p, literal_p->prop.length);
+    local_name_p = parser_new_ecma_string_from_literal (literal_p);
```

Comment:
Due to this change, the `lexer_literal_t *literal_p` local variable can be dropped. Similar cases below.